packages/network-of-terms-catalog/catalog/queries/lookup/rkdartists.rq

```diff
@@ -1,27 +1,69 @@
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
-PREFIX schema: <http://schema.org/>
+PREFIX schema: <https://schema.org/>
 PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
 
 CONSTRUCT {
     ?uri a skos:Concept ;
-        skos:prefLabel ?schema_name ;
-        skos:altLabel ?schema_alternateName ;
-        skos:scopeNote ?schema_description ;
-        rdfs:seeAlso ?rdfs_seeAlso .
+        skos:prefLabel ?prefLabel ;
+        skos:altLabel ?altLabel ;
+        skos:scopeNote ?scopeNote ;
+        rdfs:seeAlso ?seeAlso ;
+        skos:label ?label .
 }
 WHERE {
     # For example:
     # Rembrandt: <https://data.rkd.nl/artists/66219>
     # Vincent van Gogh: <https://data.rkd.nl/artists/32439>
     VALUES ?uri { ?uris }
+    {
+        SELECT ?uri
+            (GROUP_CONCAT(DISTINCT ?schema_description; separator="; ") as ?descriptions)
+            (GROUP_CONCAT(DISTINCT ?birthPlace; separator="/") as ?birthPlaces)
+            (GROUP_CONCAT(DISTINCT ?deathPlace; separator="/") as ?deathPlaces)
+            (GROUP_CONCAT(DISTINCT ?additionalType; separator=", ") as ?additionalTypes)
+        WHERE {
+            ?uri a ?type .
+            VALUES ?type { schema:Person schema:Organization }
 
-    ?uri a ?type .
-    VALUES ?type { schema:Person schema:Organization } .
+            ?uri ?name ?label .
+            VALUES ?name { schema:name schema:alternateName }
 
-    OPTIONAL { ?uri schema:name ?schema_name . }
-    OPTIONAL { ?uri schema:alternateName ?schema_alternateName . }
-    OPTIONAL { ?uri schema:description ?schema_description . }
+            OPTIONAL { ?uri schema:description ?schema_description . }
+            OPTIONAL { ?uri schema:birthPlace ?birthPlace .
+                FILTER(langMatches(lang(?birthPlace),"en")) }
```

Comment:
@RubenSchalk Is the `en` intentional? Shouldn’t we prefer Dutch information by default?

Reply:
@ddeboer sdo:Description zit er enkel als @en in zie ik, dus dat zal de overweging zijn geweest om alle tags in @en te doen (vermoed ik). Laten we het maandag meenemen in ons overleg over meertaligheid en hoe daarom mee om te gaan.

Reply:
`schema:Description` is indeed only available in English, but that one is isn’t filtered in the query.

The other fields, `schema:description`, `:birthplace`, and `:deathPlace` _are_ filtered but available in Dutch too, so I switched to `"nl"`.